presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/db/H2DaoProvider.java

```diff
@@ -31,8 +32,10 @@ public H2DaoProvider(DbResourceGroupConfig config)
     {
         JdbcDataSource ds = new JdbcDataSource();
         ds.setURL(requireNonNull(config.getConfigDbUrl(), "resource-groups.config-db-url is null"));
-        DBI dbi = new DBI(ds);
-        this.dao = dbi.open(H2ResourceGroupsDao.class);
+        this.dao = Jdbi.create(ds)
+                .installPlugin(new SqlObjectPlugin())
+                .open()
```

Comment:
Does the `Handle` need to be closed?

Reply:
Yes, this was broken previously. Will fix.

Reply:
It turns out the tests rely on this behavior because they use an anonymous in-memory database. Added a TODO.